example/jvm/src/main/scala/example/Server.scala

```diff
@@ -50,19 +53,23 @@ object Server extends SimpleRoutingApp with Api{
       } ~
       post {
         path("api" / Segments){ s =>
-          extract(_.request.entity.asString) { e =>
+          extract(_.request.entity match {
```

Comment:
@lihaoyi I must be missing an import; that doesn't work for me.
I searched all over the source and couldn't find an equivalent to the old `.asString` method.